.github/dependabot.yml

```diff
@@ -3,7 +3,9 @@
 version: 2
 updates:
   - package-ecosystem: "github-actions"
-    directory: "/"
+    directories:
+      - "/workflows"
```

Comment:
just wondering if we shoudl keep this one at `"/"` because that's the one explicitly described in the docs.

Reply:
Yes, we can do that. Also I think these folders need to be from root, since the documentation for` /` states it maps `/.github/workflows`

Reply:
Pushed. PTAL